src/main/java/org/aksw/simba/lemming/creation/Inferer.java

```diff
@@ -274,13 +249,13 @@ public void iterateStmts(Model newModel, Model sourceModel, OntModel ontModel, M
 	 * @param filePath path to the ontology file
 	 * @return OntModel Object
 	 */
-	public OntModel readOntology(String filePath, String base) {
+	private OntModel readOntology(String filePath, String base) {

```

Comment:
The term "base" is misleading here. With "base", the jena framework typically means the base IRI of a file. However, it looks like it is used here as a language identifier. Please rename the variable (and add it to the javadoc comment).

Reply:
I changed the name to "fileType".